docs/quickstart.rst

```diff
@@ -1,6 +1,68 @@
 Quickstart
 ==========
 
+Use *septentrion* as your migration tool in 2 easy steps:
+  1. Write your migrations.
+  2. Run your migrations.
```

Comment:
3. Draw the rest of the owl :D
![](https://cdn2.manhattanprep.com/gmat/wp-content/uploads/sites/18/2018/09/rm-3-image-1.png)

(note: I was the one writing this initially)